api/generic-cti/answer-event.rst

```diff
@@ -0,0 +1,206 @@
+Call answered
```

Comment:
Proposal for the headline: Event: answer

Reply:
What do you think, Gerrit?

Reply:
In case leave it as it is.

Reply:
I prefer the "Call answered", not because it provides a better headline for the article, but it provides a better bullet point for the menu on the left.

Reply:
Okay, we'll leave it as is then :)